tutorials/getting_started/rules_advanced.md

```diff
@@ -171,18 +268,51 @@ Also notice a line was added to `lightsOut` to delete the entry in the `cache` w
 That will cause the rule to create a new timer the next time the rule runs.
 It could be coded to reuse the Timer instead which is an exercise for the reader.
 
+:::
+
+::: tab JRuby
+
+In JRuby, an easy way to reschedule the same timer is done by providing a unique `id` to the timer.
+This is called a [reentrant timer](https://openhab.github.io/openhab-jruby/main/OpenHAB/DSL.html#reentrant-timers).
+
+The most convenient ID to use is the Item object for which the timer is operating, but you can use anything as the ID, e.g. a String, a number, the rule uid, etc.
+
+```ruby
+logger.info "Motion was detected"
+FrontPorchLight.on
+
+after 30.minutes, id: FrontPorchLight do |timer|
+  logger.info "No more motion, turning off the light"
+  timer.id.off # We can do this because the Timer's id was set to the item object
+               # It is the same as FrontPorchLight.off
+end
+```
+
+While it may seem straightforward, the JRuby helper library manages the timer and rescheduling internally to reduce the need for repetitive code.
```

Comment:
```suggestion
While it may not seem straightforward, the JRuby helper library manages the timer and rescheduling internally to reduce the need for repetitive code.
```

Is this what you meant?

Reply:
I'm trying to say that the user code seems simple (straightforward), so I did mean to say "While it may seem straightforward" here. 

I will remove that whole clause to simplify the sentence.